Mods/BreathOfTheWild_Intel Fix Flipped Lights/rules.txt

```diff
@@ -0,0 +1,6 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+name = Fix Flipped Intel Lights"
+path = "The Legend of Zelda: Breath of the Wild/Graphics/Intel/Remove Flipped Intel Lights"
+description = Fixes flipped lights common in 23.xx and older Intel drivers. -Altros

```

Comment:
Basically, this should probably have the same description as the other rules.txt file.

Reply:
Moved credits and pack to Workarounds/Intel iGPU